src/main/java/de/retest/web/selenium/NoElementWithHighEnoughMatchFoundException.java

```diff
@@ -8,6 +9,9 @@
 
 	public NoElementWithHighEnoughMatchFoundException( final Element resultFromExpected ) {
 		super( "The element " + resultFromExpected
-				+ " from the Golden Master could not be matched with high enough confidence." );
+				+ " from the Golden Master could not be matched with high confidence > "
+				+ System.getProperty( Properties.ELEMENT_MATCH_THRESHOLD_PROPERTY )
+				+ ". To change required confidence, please set property "
+				+ Properties.ELEMENT_MATCH_THRESHOLD_PROPERTY );
```

Comment:
Since this is a exception for the user and not for us, should the exception contain following points?

1. Cause: "The element changed too much"
2. Conclusion: "The element 'input' from the Golden Master could not be matched with high enough confidence." 		
3. Solution: "Please adjust the property"

Reply:
Hmm. If we were certain that it's because the element changed too much, we could just go on and match it. The thing is ... it might be a different element altogether. So I think the current message is good. I'll improve wording a bit, but I won't change it drastically...